README.md

```diff
@@ -0,0 +1,28 @@
+# Docker Devbox with WSL2
+You know the Docker Devbox with Vagrant.
+
+If you want learn more on Docker Devbox, you can read this article: [Docker Devbox](https://github.com/inetum-orleans/docker-devbox)
+
+But now, with WSL2, you can have a Docker Devbox without the overhead of a VM. This is a simple and lightweight setup to get you started.
+
+Less space allowed, less memory used, less CPU used, less overhead.
+
+And you don't even have to install Docker Desktop and therefore without license limitation linked to Docker Desktop.
+
+## Requirements
+WSL2 with Ubuntu and check if git is installed, if not install it.
+
+## Installation
+Clone this project inside your WSL and run the following command in the project directory:
+```bash
+git clone https://github.com/inetum-orleans/docker-devbox-wsl.git
+
+cd docker-devbox-wsl
+
+./install.sh
+```
```

Comment:
Note for later (because we need to ensure this works first), we could rewrite it like this instead:

```suggestion
Run this command to download the script and run it as root:
```bash
curl https://raw.githubusercontent.com/inetum-orleans/docker-devbox-wsl/main/install.sh | sudo bash
```
```